examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -142,13 +146,36 @@ function default_cache(Y, params, spaces, numerics, simulation)
             Operators.StencilCoefs{-half, half, NTuple{2, FT}},
         ),
         params,
-        ᶠupwind_product = upwinding_mode == :first_order ? ᶠupwind_product1 :
-                          upwinding_mode == :third_order ? ᶠupwind_product3 :
-                          nothing,
+        energy_upwinding,
+        tracer_upwinding,
         ghost_buffer = ghost_buffer,
     )
 end
 
+# TODO: All of these should use dtγ instead of dt, but dtγ is not available in
+# the implicit tendency function. Since dt >= dtγ, we can safely use dt for now.
```

Comment:
I'm not sure I follow this comment here. What does it mean that since "dt >= dtγ, we can safely use dt for now"? Wouldn't be the opposite more safe? I.e., using the smaller of the two, to make sure that we do not incur in CFL violations?

Reply:
If I understand correctly, using a larger dt means that the effect of FCT is larger to compensate for the higher Courant number. Also, @Zhengyu-Huang confirmed that we can use dt here instead of dtγ.